### DISCUSSION TITLE
Welcome to blueseer Discussions!

### DISCUSSION THREAD

**Discussion by vaughnte** [blueseerERP/blueseer#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by skarlsrud** [comment]
I am just taking my first looks at Blueseer as a manufacturing business owner and have some questions I hope someone can help me with since I can't find this type of info on the web site or docs.  We are evaluating Blueseer to determine if we should build upon our existing relational database or dump that and move to an ERP.
1.  What is the recommended system config (ram, hd, processor)?  I am installing this on an AWS Windows server.
2. We need a multi user system (10-20 simultaneous users) that are both on Macs and PCs.  What is the user interface?  Web  browser?
3. Are there consultants that specialize in Blueseer that can help us learn about the program and then determine if it is the right solution for my company?
Thank you,
Steve
